main.nf

```diff
@@ -1148,6 +1177,62 @@ process split_normalize {
 
 }
 
+/////////////// Collect QC into single file ///////////////
+
```

Comment:
Regular sample QC and mito QC jsons are collected and merged into final .QC at this point